Port over the FAQ page from hedgedoc.org to docs.hedgedoc.org (#1532)
hedgedoc/hedgedoc: docs/content/faq.md

In the meantime you can use your browsers print to PDF Feature.
This [page](https://www.digitaltrends.com/computing/how-to-save-a-webpage-as-a-pdf/) explains how to do that for multiple browsers.

## Why can't I embed some PDFs?  
Many servers don't allow the embedding of their content on arbitrary sites.

For a more technical explanation:  
The `X-Frame-Options` header can be used to specify if a given webpage can be embedded.
For security reasons this header is often set to `SAMEORIGIN`, which disallows embedding on other origins.
To be able to embed a PDF inside a HedgeDoc note, the server that hosts the PDF must either send no `X-Frame-Options`
header (which might be insecure) or include the URI of your HedgeDoc instance in an `ALLOW-FROM` statement.
See [Mozillas docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options) for more details.  
Also note that the `X-Frame-Options` header [is being obsoleted](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors)
by the `frame-ancestors` statement in the `Content-Security-Policy` header.

> ## Why can I not embed a HedgeDoc note in other pages using iframes?

Reviewer: ```suggestion
## Why can't I embed a HedgeDoc note in other pages using iframes?
```

Making it consistent with the previous question

Allowing your HedgeDoc instance to be embedded in other pages increases the risk of [clickjacking](https://en.wikipedia.org/wiki/Clickjacking),
[XSS](https://en.wikipedia.org/wiki/Cross-site_scripting) and other attacks.
Therefore, **we recommend to not enable** this option.  
If you still want to allow embedding via iframe, ensure that:

- Your HedgeDoc instance is served via HTTPS

> - `cookiePolicy` / `CMD_COOKIE_POLICY` is set to `none` (Otherwise you will get a `AUTH failed: No cookie transmitted` error.)

Reviewer 2: Maybe we should link the corresponding part of the config docs here?
